pkg/registry/extensions/rest/storage_extensions.go

```diff
@@ -47,6 +47,25 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 	return apiGroupInfo, true
 }
 
+// Wraps deploymentstore.DeploymentStorage to override RollbackREST.ProducesObject
+type DeploymentStorage struct {
```

Comment:
There's no need to add this type.

Reply:
You're right. I was worried about deploymentStorage being used elsewhere and getting updated. But it doesn't have member functions and isn't used elsewhere. Updated

Reply:
Something happened (rebase?) and this showed up again?